### ISSUE TITLE
Released App crashes upon receiving notification

### ISSUE THREAD

**Issue by iarmankhan** [cryptoticket/react-native-push-notification#5]
Hey, it's Arman again.

So after implementing this, everything was working correctly in development mode, I created a release version of app using following tutorial.
[React native Publish apk](https://reactnative.dev/docs/signed-apk-android)

When sending notifications using this API https://fcm.googleapis.com/fcm/send caused the APP to crash with following log
[Log File](https://del.dog/ysybustali)

This was working in development and still working in development.

Sending notifications from firebase console worked only for the first time, after that there were no new notifications received.

I also noticed that notifcation does not create any sound in release version whilst it does in development version.

**Comment by ryzhak** [comment]
@iarmankhan 

1. please post here the notification data that crashes the app

2. check the glide issue https://github.com/cryptoticket/react-native-push-notification#troubleshooting in troubleshooting section, may be it can help

**Comment by iarmankhan** [comment]
1. 
```
data: {
		title: "Hello World",
		message: "First notification",
	}
```
This was the data, but I've tried using different data too! Also this still works on development version (while server is running)
Also whenever I send notification , the release app closes itself without any log, I got log after trying to send notifications multiple times [Latest crash log](https://del.dog/xugucylygh.txt)

2 I'll try that and inform you with the results.

**Comment by iarmankhan** [comment]
Hello @ryzhak 

After my last response, I added glide issue solution in gradle and created a new release for the app. 
I noticed some issues, so conducted many tests myself on it. 
Here is a link for album that contains test notifcations, [Album with tests](https://imgur.com/a/RsDh2jK)
I've added the screenshots with the situation in description. give this a read.

From these all tests I can say the problem is with Event template. As I couldn't sent any notifications with FCM API that contains media, also I noticed that when I send notification from firebase console it uses Android default template (which in my opinion is best) for expandable image.

I'll try to change some code in those kotlin files and inform you with results.
